From 873217674cc824b4c1cfdad4867c560c60e8d806 Mon Sep 17 00:00:00 2001
From: Jake Waksbaum <jake.waksbaum@gmail.com>
Date: Fri, 18 Jan 2019 16:56:04 +0000
Subject: [PATCH] Replace nose-parameterized with parameterized

According to https://pypi.org/project/nose-parameterized
nose-parametrized is deprecated in favor of parameterized.
---
 test-requires.txt | 2 +-
 test.py           | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/test-requires.txt b/test-requires.txt
index ed8195f..3621aa6 100644
--- a/test-requires.txt
+++ b/test-requires.txt
@@ -1,2 +1,2 @@
 nose==1.3.0
-nose_parameterized==0.3.3
+parameterized==0.6.1
diff --git a/test.py b/test.py
index 966c9de..45a0075 100644
--- a/test.py
+++ b/test.py
@@ -5,7 +5,7 @@
 import textwrap
 
 from nose.tools import assert_equal
-from nose_parameterized import parameterized, param
+from parameterized import parameterized, param
 
 sys.path.append("pp/")
 import pp
